src/model.rs

```diff
@@ -724,6 +724,156 @@ impl Collections {
         }
     }
 
+    /// Forbid pickup on last stop point of vehicle journeys and forbid dropoff
+    /// on first stop point of vehicle journeys.
+    ///
+    /// However, there is an exception to this rule for authorized stay-in
+    /// between vehicle journeys. It is possible to get in the last stop point
+    /// of a vehicle journey or get out on the first stop point of a vehicle
+    /// journey, if and only if the 2 stop points are different and times do not
+    /// overlap.
+    ///
+    /// WARNING: The current implementation does not handle stay-in for vehicle
+    /// journeys with different validity patterns.
+    ///
+    /// Here is examples explaining the different situations of stay-in
+    /// situations (for pick-up and drop-off, XX means forbidden, ―▶ means
```

Comment:
```suggestion
    /// Here are examples explaining the different stay-in situations
    /// (for pick-up and drop-off, XX means forbidden, ―▶ means
```

Reply:
Done in 0e5e40408a6ab5201738c7c4c344dde909a9ac04.